sandbox/index-local.html

```diff
@@ -56,6 +56,12 @@
                 </a>
                 <a href="javascript:void(null);" id="btnInspector" class="hidden"><img src="./Assets/Icon_EditModel.svg"
                         alt="Display inspector" title="Display inspector" /></a>

```

Comment:
Do not update the main bar. I prefer to keep everything UI related into the inspector

Reply:
No extra button in sandbox, but a callback when the animation scene is loaded that updates the animation group drop down and co. All right.

Reply:
No extra button done in https://github.com/BabylonJS/Babylon.js/pull/7252/commits/7c892e0aae5ae7606fa23989738d49c61e182b6b

Callback coming next